src/main/java/seedu/duke/DialogBox.java

```diff
@@ -0,0 +1,54 @@
+package seedu.duke;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+import java.io.IOException;
+
+
+public class DialogBox extends HBox {
+
+    @FXML
+    private Label dialog;
+    @FXML
+    private ImageView displayPicture;
+
+    public DialogBox(String text, Image img) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/DialogBox.fxml"));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Could there be a better way of handling this exception instead of just printing the stack trace?